docs/getting-started/anti-phishing.md

```diff
@@ -2,28 +2,28 @@
 
 Most emails on Internet are spams and phishing attempts. To not forward unwanted and illegitimate emails, SimpleLogin automatically check the sender authenticity before forwarding the email to your mailbox.
 
-The authentication check is currently based on [DMARC](https://en.wikipedia.org/wiki/DMARC) which is then based on [SPF](https://en.wikipedia.org/wiki/Sender_Policy_Framework) and [DKIM](https://en.wikipedia.org/wiki/DomainKeys_Identified_Mail), the 2 popular email authentication methods.
+The authentication check is currently based on [DMARC](https://en.wikipedia.org/wiki/DMARC), which is then based on [SPF](https://en.wikipedia.org/wiki/Sender_Policy_Framework) and [DKIM](https://en.wikipedia.org/wiki/DomainKeys_Identified_Mail), the two popular email authentication methods.
 
 When the authentication check fails, depending on the policy set by the sender, an email can either:
 
-- be put into **quarantine**. This happens when the sender explicitly states their emails should respect DMARC and any email that violates this should either be quarantined or rejected. In this case, SimpleLogin will inform you through email and send a [notification](https://app.simplelogin.io/dashboard/notifications) and you can download the quarantined email. Unfortunately the only solution is the sender fixing their DMARC setting as their emails are probably being refused or put into Spam everywhere.
+- be put into **quarantine**. This happens when the sender explicitly states their emails should respect DMARC and any email that violates this should either be quarantined or rejected. In this case, SimpleLogin will inform you through email and send a [notification](https://app.simplelogin.io/dashboard/notifications), and you can download the quarantined email. Unfortunately, the only solution is the sender fixing their DMARC settings as their emails are probably being refused or put into Spam everywhere.
 
-- have the following **warning added**. This happens when the authentication fails but the sender doesn’t set any specific action. The email can be a phishing attempt and it’s important for SimpleLogin to inform you about the potential risk. Unfortunately, false positives do happen, especially when the sender incorrectly set up their DMARC. It would be beneficial to inform the sender in this case as their emails might end up in Spam everywhere.
+- have the following **warning added**. This happens when the authentication fails but the sender doesn’t take any specific action. The email can be a phishing attempt, and it’s important for SimpleLogin to inform you about the potential risk. Unfortunately, false positives do happen, especially when the sender incorrectly sets up their DMARC. It would be beneficial to inform the sender in this case, as their emails might end up in Spam everywhere.
```

Comment:
the sender actually can't take any action but rather set a policy on what action they want the receiver to take.

Reply:
I see. Just changed it back